src/main/resources/edu/cornell/kfs/fp/document/datadictionary/DisbursementVoucherDocument.xml

```diff
@@ -148,6 +148,7 @@
     			<entry key="Purchasing" value-ref="DisbursementVoucherDocument-RoutingType-Campus"/>
     			<entry key="Account" value-ref="RoutingType-AccountingDocument-Account-sourceOnly"/>
     			<entry key="AccountingOrganizationHierarchy" value-ref="RoutingType-AccountingDocument-OrganizationHierarchy-sourceOnly"/>
+    			<entry key="ObjectCode" value-ref="RoutingType-AccountingDocument-ObjectCode-sourceOnly"/>

```

Comment:
What is the current plan for updating the various doc types to include this new ObjectCode route node? Would you prefer using the workflow XML provided under the base financials `/kfs-core/src/main/resources/workflow-upgrades/2021-12/` folder instead of SQL, and if so should that file or its contents be included in the cu-kfs project?

Reply:
Excellent question. Functionals would like to first better understand the ObjectCode routing and then determine te best time to implement locally. There is a jira in the next Sprint for this: https://jira.cornell.edu/browse/KFSPTS-24999.
Since the routing of the different documents has in many cases been customized for Cornell I am not sure we can use the XML provided by KualiCo but as a reference. I expect we need to start from the Cornell Specific workflow xml and add the new ObjectCode routing. I thought that we would have to first export the Cornell doc type, add the object code routing and then re-ingest. I tried this for the REQS doc type. Please let me know if you have some suggestion on how to better handle this or if you see any issue with the current plan. Thank you!

Reply:
Thanks for the explanation. The follow-up ticket and current plan sound good, and if we uncover issues we can address them in that future ticket.